### DISCUSSION TITLE
Unable to download

### DISCUSSION THREAD

**Discussion by xjkken007** [aclap-dev/video-downloadhelper#683] [Help and Q&A]
### Your question

Unable to download the H5 vedio

### Addon & CoApp version and details

{
  "version": "9.0.1.22",
  "target": "google",
  "channel": "stable",
  "page_url": "https://elearning.chinaacc.com/xcware/video/h5video/videoPlayhls.shtm?cwareID=54950&videoID=311&startTime=15:43&identity=3060",
  "error": {
    "details": "SideDownload error: : [hls @ 00000286ad577380] Error when loading first segment 'https://ssec2.chinaacc.com/vod5ts/CL1xtZNi3NDBxIIvtVDe5pDJGdEIVtIY/9UlzRuTlZKvekPTMJEL-HRFfHZH8Hq.sbtH-eJV2YjsfSDwdfr9aTpiZTN3731SUvi2LjSSF09zV8w0-PJOA.w__.ts?cwareID=54950&videoID=311&userID=64511363&time=1716733936875&code=614d178364e3c58b1b6b66687ec2d793&host=inner.chinaacc.com&sid=&th=pc&_pVer=1.0&_vodrs=102&vodnSid=d2420568c9da456ebcdde72f5ae2a985&idx=1'\r\n[in#0 @ 00000286ad5771c0] Error opening input: Invalid data found when processing input\r\nError opening input file https://ssec2.chinaacc.com/vodn/ssec.chinaacc.com/1716733936875/YuC6RK82SdL7JD4TDqdfoHBAcfhcL5kDZSFijVvVaP4hETgvQXCPnXxmoXaujHqu.mp4/index.m3u8?cwareID=54950&videoID=311&userID=64511363&time=1716733936875&code=614d178364e3c58b1b6b66687ec2d793&host=inner.chinaacc.com&sid=&th=pc&_pVer=1.0.\r\nError opening input files: Invalid data found when processing input\r\n",
    "downloadable_id": "downloadable_2732690989463701",
    "error": "coapp_failure",
    "id": "coapp_failure_817e916a-7c7a-4623-8491-316478ba45cb"
  }
}

**Comment by Mattmjs** [comment]
Another "unable to read key file".
